rdkit/Chem/UnitTestRegistrationHash.py

```diff
@@ -276,13 +276,15 @@ def test_enhanced_stereo_canonicalizer(self):
             ),
         )
         for mols_smis in groups:
-            print()
             csmis = set()
             for smi in mols_smis:
                 mol = Chem.MolFromSmiles(smi)
-                csmi = Chem.CanonicalizeEnhancedStereo(mol)
+                Chem.CanonicalizeEnhancedStereo(mol)
+                csmi = Chem.MolToCXSmiles(mol)
```

Comment:
As we discussed yesterday, you don't need the call to `CaonicalizeEnhancedStereo()`: `MolToCXSmiles()` takes care of that for you:
```suggestion
                csmi = Chem.MolToCXSmiles(mol)
```